thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -69,7 +79,11 @@ export default Ember.Component.extend({
 
   comment: Ember.computed('anomaly', function () {
     const attr = this.get('anomaly').attributes;
-    return attr.comment[0];
```

Comment:
wondering if doing `const attr = this.get('anomaly.attributes.comment.firstObject')` would work here, 
so we can just `return attr || [];`